build.gradle

```diff
@@ -31,7 +29,8 @@ ext {
       spock: 'org.spockframework:spock-core:0.7-groovy-2.0',
       truth: 'com.google.truth:truth:0.30',
       mockWebServer: 'com.squareup.okhttp3:mockwebserver:3.5.0',
-      okhttpLoggingInterceptor: 'com.squareup.okhttp3:logging-interceptor:3.5.0'
+      okhttpLoggingInterceptor: 'com.squareup.okhttp3:logging-interceptor:3.5.0',
```

Comment:
this should be 3.6.0 as well....feel free to leave it if you want...i can make this an okhttp version var